app/models/models.scala

```diff
@@ -1,6 +1,8 @@
 package models
 
-import org.joda.time.DateTime
+import org.joda.time._
+import com.gu.scanamo._
+import com.amazonaws.services.dynamodbv2.model.ScalarAttributeType._
```

Comment:
This is unnecessary. I think it only serves for using AWS attribute types (e.g. `val fooTableResult = LocalDynamoDB.createTable(client)("foo")('dateTime -> S)` where `S` is `String`) which we are not.